java/IBAutomater/src/ibautomater/Common.java

```diff
@@ -71,6 +72,18 @@ public static JToggleButton getToggleButton(Container container, String text) {
         return null;
     }
 
+    public static JRadioButton getRadioButton(Container container, String text) {
+        ArrayList<Component> buttons = new ArrayList<>();
+        Common.loadComponents(container, JRadioButton.class, buttons);
+        for (Component component : buttons) {
+            JRadioButton button = (JRadioButton)component;
+            String buttonText = button.getText();
+            if (buttonText == null || !button.getText().equals(text)) continue;
```

Comment:
nit - could directly use `buttonText` instead of getting it again `button.getText()`. Maybe could use case in-sensitive comparison, I think `equalsIgnoreCase()`?

Reply:
Yeah, should be `!buttonText.equals(text)`, will fix in the other methods too. 👍